Updated the website a bit.

Youngestdev opened the pull request.

samswag commented:
Could you explain what you did and next time,  you need to put more details in your commit. I think at this post we should focus more of optimization, SEO and adding features to the site, content will come later.

Youngestdev commented:
Well. i deleted the default posts and removed pages that aren't useful. Talking of SEO for now is void, we should make the website okay before planning and features ? well, idk about any feature yet. let the website be a blog roll or do you intend adding other stuffs apart from disbursing information ?

Youngestdev commented:
Added a search bar to check for posts
